plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/model/CodeWhispererModel.kt

```diff
@@ -130,6 +134,16 @@ data class SessionContext(
     var hasAccepted: Boolean = false
 ) : Disposable {
     private var isDisposed = false
+    init {
+        project.messageBus.connect().subscribe(
```

Comment:
should we pass `this` as the disposable parent?

Reply:
yes, definitely, thanks for catching this.